### PR TITLE
feat: grafana add p99, apiserver, tablet query

### PR DESCRIPTION
**Pull request by vagetablechicken**
- p99, apiserver, tablet query prc metrics
- overview expand, others collapse
- delete "deploy query count total"
- multi-value for variable "db", "deployment"

### CODE REVIEW

**Comment by aceforeverd** [comment]
make all rows (except the first row 'OpenMLDB Cluster Overview') in dashboard default closed, like original
Screenshot 2023-11-23 at 13 26 11

**Comment by aceforeverd** [comment]
May have more detailed name for the two row? 
Screenshot 2023-11-23 at 13 28 04

Separate 'Tablet Detail' in different panel for different tablet endpoint may better. Also API server may have multiple instance

**Comment by aceforeverd** [comment]
db & deployment variable should support multi-value

Screenshot 2023-11-23 at 13 49 55

**Comment by aceforeverd** [comment]
Remove 'Deploy Query Count Total' in OpenMLDB Cluster Overview

**Comment by vagetablechicken** [comment]
> May have more detailed name for the two row? Screenshot 2023-11-23 at 13 28 04
> 
> Separate 'Tablet Detail' in different panel for different tablet endpoint may better. Also API server may have multiple instance

Each apiserver will be an `instance`, no need to seprate
And I want `Tablet Detail` to be the one, so we could see the difference in one panel, don't need to check multi panels to know some tablet is slow or heavy

**Comment by aceforeverd** [comment]
> > May have more detailed name for the two row? Screenshot 2023-11-23 at 13 28 04
> > Separate 'Tablet Detail' in different panel for different tablet endpoint may better. Also API server may have multiple instance
> 
> Each apiserver will be an `instance`, no need to seprate And I want `Tablet Detail` to be the one, so we could see the difference in one panel, don't need to check multi panels to know some tablet is slow or heavy

I still think there is too many series on single graph. To compare perf among tablets, then separate with different tp value ?

File: openmldb_mixin/openmldb_dashboard.json [hunk 1]
```diff
@@ -1577,21 +1514,21 @@
                 "uid": "${DS_PROMETHEUS}"
               },
               "exemplar": true,
-              "expr": "openmldb_info_schema_deploy_response_time_seconds_count{job=~\"$job\", deploy_path=~\"$deploy_path\"}",
+              "expr": "sum({__name__=~\".*_deployment_count\", db=\"$db\", deployment=\"$deployment\"})",
```
Comment (aceforeverd) [inline openmldb_mixin/openmldb_dashboard.json:1517]: variables may be a list, always use `=~` to filter,

File: openmldb_mixin/openmldb_dashboard.json [hunk 2]
```diff
@@ -3958,16 +4228,40 @@
...
           "uid": "${DS_PROMETHEUS}"
         },
-        "definition": "label_values(openmldb_info_schema_deploy_response_time_seconds_count, deploy_path)",
-        "description": "{db_name}}_{{deploy_name}} for a deployment name",
+        "definition": "label_values({__name__=~\".*_deployment_count\"},db)",
+        "description": "{{db_name}} for a deployment name",
         "hide": 0,
-        "includeAll": true,
-        "label": "Deploy Path",
+        "includeAll": false,
```
Comment (aceforeverd) [inline openmldb_mixin/openmldb_dashboard.json:4234]: includeAll -> true

**Review by aceforeverd — CHANGES_REQUESTED** [review]

**Pull request merged**